### PR TITLE
[umoci] Initial integration

### PR DESCRIPTION
**Pull request by AdamKorcz**

### CODE REVIEW

**Comment by AdamKorcz** [comment]
@cyphar I have added initial integration of Umoci with a few fuzzers to get things started. As we add more, we will add these to continuous fuzzing as well. Which Umoci maintainers email could we add to the project.yaml? This is for receiving potential bug reports. Only email addresses added to this list will have access to crashes.

**Comment by cyphar** [comment]
Thanks. Please add all of the maintainers in [the MAINTAINERS file](https://github.com/opencontainers/umoci/blob/master/MAINTAINERS).

**Comment by AdamKorcz** [comment]
@jonathanmetzman @inferno-chromium Ready for review.

**Comment by AdamKorcz** [comment]
Thanks @jonathanmetzman

File: projects/umoci/fuzz.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package casext
```
Comment (jonathanmetzman) [inline projects/umoci/fuzz.go:1]: This file needs a license.

File: projects/umoci/reader_fuzzer.go [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package hardening
```
Comment (jonathanmetzman) [inline projects/umoci/reader_fuzzer.go:1]: This file needs a license.

**Comment by inferno-chromium** [comment]
Lets not merge this unless security impact is clear, can you please move this to draft.

**Pull request closed without merging**